sidebar.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/sidebar.css" />
+    <link rel="stylesheet" href="styles/main.css" />

```

Comment:
This section of the code if you are linking to main.css it should be "...href="./assets/main.css" but if you are linking to sidebar.css it should come last and be "...href="./assets/sidebar.css".

Reply:
Good catch, I've made the change and committed.